tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1406,3 +1432,48 @@ def test_verify_attribute_of_users_are_updated(session, ldap_data, ldap_tear_dow
         assert ldap_data['ldap_user_name'] in user_values['user']['firstname']
         assert ldap_data['ldap_user_name'] in user_values['user']['lastname']
         assert ldap_data['ldap_user_name'] in user_values['user']['mail']
+
+
+@tier2
+def test_verify_group_permissions(
+    session, auth_source_ipa, multigroup_setting_cleanup, groups_teardown, ldap_tear_down
+):
+    """Verify group permission
+
+    :id: 7e2ef59c-0c68-11eb-b6f3-0c7a158cbff4
+
+    :Steps:
+        1. Create two usergroups and link it with external group having a
+        user in common
+        2. Give those usergroup different permissions
+        3. Try login with the user common in both external group
+
+    :expectedresults: Group with higher permission is applied on the user
+    """
+    sat_admins, sat_users = settings.ipa.groups.replace(" ", '').split(',')
+    _, idm_user = settings.ipa.group_users.replace(" ", '').split(',')
+    auth_source_name = 'LDAP-' + auth_source_ipa.name
```

Comment:
Please use f-string

Reply:
@akhil-jha I think we should create a fixture/function which would return us values of `sat_admins, sat_users, idm_user, auth_source_name, user_permissions, katello_role ` so that we won't have to add this code to each test. I guess @omkarkhatavkar has also written a test which needs these values. See https://github.com/SatelliteQE/robottelo/pull/8045/files

Reply:
We both are working tests which need similar data and once anyone's PR get merged we will rebase it